src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.data;
+
+public class User {
+
+    private String email;
+    private String aboutMe;
+
+    public User(String email, String aboutMe) {
+        this.email = email;
+        this.aboutMe = aboutMe;
+    }
+
+    public String getEmail(){
```

Comment:
It's a good idea to have a setter method as well for these two. setEmail(...) and setAboutMe(...). it may be needed for future.